src/pages/Login.vue

```diff
@@ -203,14 +205,18 @@ export default {
               })
             )
             .catch((err) => {
-              this.$toasted.global.error_post({
-                message: err.message,
-              });
+                this.$toasted.global.error_post({
+                  message: err.message,
+                });
             });
         })
         .catch((err) => {
           this.loading = false;
-          this.$toasted.global.error_post({ message: err.message });
+          if (err.message == "The email address is already in use by another account.") {
```

Comment:
Instead of doing a string match on error message, use the error code.
Check out the reference docs - https://firebase.google.com/docs/reference/js/firebase.auth.Auth#createuserwithemailandpassword

Reply:
Oh and always use === for equality check (lint suggestion)